src/Components/FlashBanner.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { Banner, Sans } from "@artsy/palette"
+
+interface Props {
+  messageCode: string
+}
+
+const textMessage: (msg: string) => React.FC = (message: string) => () => (
+  <Sans color="white100" size="3" lineHeight={1} weight="medium">
+    {message}
+  </Sans>
+)
+
+const messages: { [k: string]: React.FC } = {
+  confirmed: textMessage("Your email has been confirmed."),
```

Comment:
I presume these message keys need to correspond with the [URL parameters generated by Gravity](https://github.com/artsy/gravity/blob/master/app/controllers/confirmations_controller.rb#L3-L7) - is that correct? Otherwise, we'll never surface the more nuanced error cases that _should_ be presented to end-users: for example, a user already having confirmed their email.

Reply:
:+1: